.circleci/config.yml

```diff
@@ -427,22 +545,90 @@ workflows:
               ignore: *release-branch-regex
             tags:
               ignore: /.*/
-      - run-windows-e2e-test:
+      - test-integration-setup-windows:
           requires:
             - build-binaries
           filters:
             branches:
               only:
                 - master
-                - /.*(windows|win)/
-      - test-integration:
+      - integration-build-windows:
```

Comment:
Probably no in this PR but, should we split the workflows so we don't have so many `filters.branch.only=master` 
At least we should differentiate between the one for the commit and the one for the merge to master

Reply:
I've filed an issue in order to work on this sometime https://okteto.atlassian.net/browse/DEV-534